### ISSUE TITLE
Eval can be replaced by getfield in several tests

### ISSUE THREAD

**Issue by julbinb** [JuliaStats/Distributions.jl#1235]
It probably does not matter much fr testing, but `getfield` would be more efficient than `eval` in getting the value of a variable/function name/type here

https://github.com/JuliaStats/Distributions.jl/blob/664ffa4110425e5858a096987781583b432902cd/test/univariates.jl#L24

here

https://github.com/JuliaStats/Distributions.jl/blob/664ffa4110425e5858a096987781583b432902cd/test/univariates.jl#L85

and here

https://github.com/JuliaStats/Distributions.jl/blob/664ffa4110425e5858a096987781583b432902cd/test/truncate.jl#L32

```julia
julia> using BenchmarkTools

julia> @btime getfield(Main, Symbol("Int"))
  0.015 ns (0 allocations: 0 bytes)
Int64

julia> @btime eval(Symbol("Int"))
  58.841 ns (0 allocations: 0 bytes)
Int64
```

**Comment by andreasnoack** [comment]
Just tried it out and, unfortunately, it doesn't seem to make a difference. Probably not too surprising in this case here because of all the JSON parsing and other `eval`s happening here as well. I'm wondering how much faster these tests would be if they were written as normal Julia source files.

**Comment by ararslan** [comment]
>  I'm wondering how much faster these tests would be if they were written as normal Julia source files.

cf. https://github.com/JuliaStats/Distributions.jl/issues/628

**Comment by julbinb** [comment]
Yes, compared to JSON parsing, this is definitely a minor thing.
